erda.yml

```diff
@@ -58,7 +58,7 @@ services:
       replicas: 1
     envs:
       DEBUG: "true"
-      MONITOR_ADDR: monitor:7096
```

Comment:
MONITOR_ADDR useless?

Reply:
already defined globally in erda.yml global envs.

Reply:
see https://github.com/erda-project/erda/blob/master/erda.yml#L13